docs/content/preview/api/ysql/the-sql-language/statements/savepoint_release.md

```diff
@@ -54,6 +54,11 @@ RELEASE [ SAVEPOINT ] name
 
 The name of the savepoint you wish to release.
 
+{{<note title="Other savepoints may be released">}}
+Releasing a savepoint also releases any savepoints created between the one you're releasing and the release command.
```

Comment:
```suggestion
When you release a savepoint, all savepoints created subsequently up to the point of the release command are also released. For example, if savepoint 'a' is created, followed by savepoint 'b', and you then release 'a', 'b is also released.
```

Reply:
I like @bllewell version more. Just a single sentence:

> When you release a savepoint, all savepoints that were created after it was created are also released.

@ddhodge yours looks a little too repetitive. We don't need an example for such simple "logic".

Reply:
Comments seem to be out of time-sequence on this page. Anyway, the "deploy preview" of the not doesn't yet show this:

> When you release a savepoint, all savepoints that were created after it was created are also released.

Make it so. And then I'll approve the PR.